pubspec.yaml

```diff
@@ -10,7 +10,7 @@ authors:
   - Trent Grover <trent.grover@workiva.com>
 homepage: https://github.com/Workiva/w_flux
 dependencies:
-  react: ">=0.5.0 <2.0.0"
+  react: ">=0.5.0 <3.0.0"
```

Comment:
this wouldn't be a breaking change:
http://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api